.github/workflows/check.yml

```diff
@@ -3,8 +3,6 @@ name: Checks
 on:
   push:
   pull_request:
-  schedule:
-  - cron: "0 0 1 * *"
```

Comment:
Did you do `git pull --rebase`? I don't understand why changes I made in a previous commit show up here. Is this a GitHub bug? This change is already ins master.

Reply:
I rebased my master onto my current branch.

I never do this usually, I nearly always just merge, so I guess I did something wrong. In the end when we know what changes we want, I can make a clean Pull Request.